backend/src/main/resources/db/migration/V1.36.0__email_on_failed_integration.sql

```diff
@@ -0,0 +1,91 @@
+--
+-- Create a new b/a/et platform/notifications/integration-failed
+--   and then link that to a default behaviour group to send email
+--   to admin
+insert into  bundles (name, display_name, id, created)
+    values ('platform','Platform', public.gen_random_uuid(), now() at time zone 'UTC' );
+
+insert into applications (bundle_id, name, display_name, id, created)
+    select b.id ,
+            'notifications',
+            'Internal',
+            public.gen_random_uuid(),
+            now() at time zone 'UTC'
+    from bundles b
+    where b.name = 'platform'
+        ;
+
+insert into event_type (application_id, name, display_name, id )
+    select a.id ,
+            'integration-failed',
+            'Integration failed',
+            public.gen_random_uuid()
+    from applications a
+    where a.name = 'notifications'
+    ;
+
+-- create the default BG with accountId = null
+insert into behavior_group (id, account_id, bundle_id, display_name, created )
+    select public.gen_random_uuid(),
+        null,
+        b.id,
+        'Integration_failed_mail',
```

Comment:
Will this be displayed in the frontend?

Reply:
I don't expect it right now, as it is in a bundle that is not explicitly shown, but it may happen in the future